plugins/ci/skills/prow-job-extract-must-gather/extract_archives.py

```
@@ -0,0 +1,202 @@
+        if size_bytes < 1024.0:
+            return f"{size_bytes:.1f} {unit}"
+        size_bytes /= 1024.0
+    return f"{size_bytes:.1f} PB"
+
+
+def extract_tar_archive(tar_path, extract_to):
+    """Extract a tar archive (including .tar.gz and .tgz)."""
+    try:
+        print(f"  Extracting: {tar_path}")
+        with tarfile.open(tar_path, 'r:*') as tar:
+            tar.extractall(path=extract_to)
```

Ruff (0.16.0)

[error] Uses of `tarfile.extractall()`

(S202)